docs/developer-docs/latest/setup-deployment-guides/typescript.md

```diff
@@ -48,7 +48,7 @@ yarn create-strapi-app@latest my-project --typescript
 
 ## Understand TypeScript support
```

Comment:
> I may be a bit late with that documentation but I'm not sure `typescript.md` is currently at the right place in the TOC 🤔 It's treated as an installation guide but that documentation isn't an installation guide. Maybe it should be in the configurations (with a link to that doc in the CLI installation guide where we mention TypeScript, explicitly saying something like "more information about TypeScript in the dedicated documentation"). Although I'm aware this doc is not 100% configuration. What do you think?

It is on the same level as the file structure and configuration documents. I don't think that is treating it as an installation guide. There isn't a great place for it, because of the way our documentation is structured. I am in favor of moving it anywhere where it is easier to find. This is the most requested feature from users, it shouldn't be hard to find the documentation.

Reply:
<img width="221" alt="Screenshot 2022-06-08 at 10 05 17" src="https://user-images.githubusercontent.com/19183360/172565074-a925c0f3-c088-4c68-a3b8-373c9171cd1f.png">

^ This is what I mean when I say it's "treated as an installation guide in the TOC". But then it's not listed in the introduction of the installation guides, and in the CLI guides there's no link to the TypeScript doc.
And again, it's a mix of installation instructions, concept explanations and configuration/customisation, which indeed makes it hard for that piece of documentation to fit perfectly anywhere but that is precisely the issue. That's why I'm pointing that it's rather confusing.
Now I'm aware it might be too late to change the whole documentation, but we could at least try to find the best place in the TOC, and make sure it's indeed easy for users to find it (hence the links in relevant docs, to increase easy access to the TypeScript doc).